kommunicate-script-example/README.md

```diff
@@ -40,18 +41,19 @@ Example:
             var h = document.getElementsByTagName("head")[0]; h.appendChild(s);
             window.kommunicate = m; m._globals = kommunicateSettings;
         })(document, window.kommunicate || {});
-
+</script>
 ```
 > **Note:** Replace the `<APP_ID>` with the `appId` copied from the <a href="https://dashboard.kommunicate.io/settings/install" target="_blank">Install Section</a> from the Kommunicate Dashboard.
 
 
 ### Make chat widget fullscreen on pageload
-To make the Kommunnicate chat widget Full screen on page-load copy & paste the following code inside the `onInit: function() { ... }` of the `kommunicateSettings` variable.
+To make the Kommunicate chat widget Full screen on page-load copy & paste the following code inside the `onInit: function() { ... }` of the `kommunicateSettings` variable.
```

Comment:
Check the paste the code inside line as well

Reply:
Do customers need to copy code inside onInit or copy whole code?

Reply:
Sentence reframed